src/Microsoft.DotNet.Darc/src/Darc/Options/GatherDropCommandLineOptions.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using CommandLine;
+using Microsoft.DotNet.Darc.Operations;
+
+namespace Microsoft.DotNet.Darc.Options
+{
+    [Verb("gather-drop", HelpText = "Gather a drop of the outputs for a build")]
+    internal class GatherDropCommandLineOptions : CommandLineOptions
+    {
+        [Option('i', "id", SetName = "rootbuild", HelpText = "Bar ID of build.")]
+        public int RootBuildId { get; set; }
+
+        [Option('r', "repo", SetName = "rootbuild", HelpText = "If set, gather a build drop for a build of this repo. Requires --commit.")]
```

Comment:
>[Option('r', "repo", SetName = "rootbuild" [](start = 8, length = 42)

Is having the same SetName here than the previous correct?

Reply:
It means you can't use the options together.  Which is fine here.